.github/workflows/linux_wheel.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Linux Wheels
+
+on:
+  pull_request:
+
+jobs:
+  generate-build-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    container:
+      image: python:3.9
+    steps:
+      - name: Clone pytorch/pytorch
+        uses: actions/checkout@v2
+      - id: set-matrix
+        run: |
+          # outputting for debugging purposes
+          python .github/scripts/generate_binary_build_matrix.py
```

Comment:
Is there a reason to run the script twice rather than running once and echoing the output twice (the second time with `::set-output`)?

Reply:
I like outputting the matrix in the first run for debugging purposes so we can see what the actual json blob looks like. perhaps it’d be better to just store that json blob as an artifact for later debugging purposes?

Reply:
I guess I just meant, why run the script twice when the output is the same? Why not instead do something like this?
```sh
MATRIX=$(python .github/scripts/generate_binary_build_matrix.py)
echo "${MATRIX}" # outputting for debugging purposes
echo "::set-output name=matrix::${MATRIX}"
```

Reply:
Oh you could potentially do that, but running the script is at a fairly low cost